lib/cursor.js

```diff
@@ -105,7 +105,8 @@ var Cursor = function(bson, ns, cmd, options, topology, topologyOptions) {
     batchSize: options.batchSize || cmd.batchSize || 1000,
     currentLimit: 0,
     // Result field name if not a cursor (contains the array of results)
-    transforms: options.transforms
+    transforms: options.transforms,
+    raw: options.raw || (typeof cmd === 'object' && cmd.raw)
```

Comment:
`typeof null === 'object'`, so we should change this to:

```js
raw: options.raw || (cmd && cmd.raw)
```

Reply:
changed in prior PRs, rebased into this one.